web/src/app/components/modal/modal.component.ts

```diff
@@ -182,19 +184,21 @@ export class ModalComponent extends BaseModal implements OnInit {
    
     this.showProgress();
     
+    const initialGender = '';
 
     this.defendantAndCaseForm = this.fb.group({
-
-      
         defendantname: new FormControl(),
         defendantrace: new FormControl(['', Validators.compose(
           [Validators.required, this.validateSelection]
         )]),
-        defendantgender: new FormControl(['', Validators.compose(
-          [Validators.required, this.validateSelection]
-        )]    ),
-      
-     
+        defendantGender: new FormControl(initialGender, {
+          validators: [
+            Validators.required, 
+            Validators.minLength(1), 
+            validateGender
```

Comment:
Validating the gender selection is an acceptable type instead of just looking for value length